webapp-django/crashstats/base/tests/test_finders.py

```diff
@@ -0,0 +1,33 @@
+import copy
+
+from nose.tools import assert_raises
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.core.management import call_command
+
+from crashstats.base.tests.testbase import DjangoTestCase
+
+
+class LeftoverPipelineFinder(DjangoTestCase):
+    """Test our custom staticfiles finder class."""
+
+    def test_missing_css_source_file(self):
+        busted_pipeline = copy.deepcopy(settings.PIPELINE)
+        # Doesn't matter which key we chose to bust
+        for key in busted_pipeline['STYLESHEETS']:
```

Comment:
It's less surprising because the code matches the intention. Using loops just to get the first item seems weird. I hit the `break`, was surprised, and then had to reread the code to understand what you were trying to do.

Is it helpful to support `busted_pipeline` values that might be generators here?